### PR TITLE
[TW-1198] Doc Runner response/request body is not available for team members

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM

**Review by postfrzero — APPROVED** [review]

File: src/pages/docs/collections/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ You can configure the Collection Runner to meet your development needs. You can
     * **Data** - A [data file](/docs/collections/running-collections/working-with-data-files/) for the collection run.
     * **Persist responses for a session** - Log the response headers and bodies so you can review them after running the collection. For large collections, persisting responses may affect performance.
 
-    > Request and response details are persisted locally during your current Postman session and aren't saved permanently. Signing out of Postman, signing into another device with the same account, or refreshing your browser will end your session and remove the logged data.
+        > Request and response details are persisted locally during your current Postman session and aren't saved permanently. Signing out of Postman, signing into another device with the same account, or refreshing your browser will end your session and remove the logged data.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collections/running-collections/intro-to-collection-runs.md:62]:
```suggestion
        > During your current Postman session, request and response details are persisted locally and not saved permanently. Your session will end and your logged data will be removed when you sign out of Postman, sign in to another device with the same account, or refresh your browser.
```

File: src/pages/docs/collections/running-collections/intro-to-collection-runs.md [hunk 2]
```diff
@@ -59,7 +59,9 @@ You can configure the Collection Runner to meet your development needs. You can
...
+        >
+        > Response and request details are available for the person who started the collection run. Other team members can't view details for collection runs that you start.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collections/running-collections/intro-to-collection-runs.md:64]:
```suggestion
        > Only the person who started the collection run can view response and request details. 
```

**Review by nerminamiller-postman — COMMENTED** [review]
@mark-anthony-postman left you a bit of feedback, more as an exercise. Feel free to reject any and all comments.

**Review by malvikach — CHANGES_REQUESTED** [review]
Can we also add how users can enable persistence for their sessions? (the configuration on runner allows it and maybe couple it with a screenshot)

**Comment by mark-anthony-postman** [comment]
> Can we also add how users can enable persistence for their sessions? (the configuration on runner allows it and maybe couple it with a screenshot)

@malvikach Already added! There is a procedure with a screenshot that shows users how to configure a collection run, including using the persist option. This updated date is part of the procedure. You should be able to see the whole procedure in the rich diff: https://github.com/postmanlabs/postman-docs/pull/4883/files?short_path=08a20f7#diff-08a20f766a7df607f27d588aac9cdb7dae0c1d0402796a9ddeb5b3b45566f6dd

**Review by malvikach — APPROVED** [review]
Looks good!

**Review by beatrixSze — APPROVED** [review]

**Pull request merged**